### ISSUE TITLE
[RFE] Allow import of EKS Cluster without having to provide AWS Access Key and Secret Key

### ISSUE THREAD

**Issue by hectoralicea** [rancher/rancher#41158]
**Is your feature request related to a problem? Please describe.**
Yes.  In our AWS GovCloud Account, they do not allow AWS Keys to be used.  We must use either MFA, IAM instance roles, or use Service Roles, which  is able to assume a role.  This will not allow me to import an EKS.  The EKS being imported is running from a Rancher on another EKS with the node instance role that has the rights to manage both cluster.  

**Describe the solution you'd like**

The better way is to not make AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY required parameter to import the cluster and allow it to use the native VM instance role that Rancher is running under.

**Describe alternatives you've considered**

If we provide any AWS Access Key and Secret Key it will override any node instance role native on the VM.  Some how if we can get into the imported cluster and remove the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY and let it use the node instance role we should be fine.

**Additional context**

My understanding is that Rancher is big in the DoD, and Intel community, but I dont see how if it's not able to support this lock down requirements for AWS in the Gov Cloud.

**Comment by hectoralicea** [comment]
Also trying to figure out if this could be done with Rancher CLI as well.  Appears so, but have not been able to get it to work yet.

**Comment by hectoralicea** [comment]
Figured it out.

Best way to import EKS cluster and not require AWS keys is to import it as a generic Kubernetes.  Make sure the following properties are set in your helm chart which is how I solved the problem

My list of issues.  
In settings the following was wrong

- Cacerts was blank.  It was not ingesting my CA certs cause I had `privateCA: false` in stead of true
- server-url did not have the https://. It only had the host name.  Make sure you Put in the entire URL
- I did not create the tls-ca-additional.  Created that and set `additionalTrustedCAs: true`
